src/scss/page/main/date.scss

```diff
@@ -0,0 +1,22 @@
+@import "~scss/_mixins";
+
+.pageMainDate {
+	.wrapper { width: 704px; margin: 0px auto; padding: 40px 0px 80px 0px; user-select: none; }
+
+	.headSimple {
+		.side.right { gap: 0px; }
+		.side.right {
+			.icon.calendar { background-image: url('~img/icon/relation/date.svg'); }
```

Comment:
Add `width: 24px !important; height: 24px !important;` to icons as withBackground class size is 28px